doc/release-notes/rl-2511.section.md

```diff
@@ -206,6 +206,8 @@
 
 - `inspircd` has been updated to the v4 release series. Please refer to the upstream documentation for [general information](https://docs.inspircd.org/4/overview/#v4-overview) and a list of [breaking changes](https://docs.inspircd.org/4/breaking-changes/).
 
+- `kanata` now requires karabiner-dk version 6.0+ or later. The package has been updated to use the new karabiner-dk package and the darwinDriver output stays at the version defined in the package.
```

Comment:
At this time, if we treat this as a normal/real breaking change, it means
- 1.10.0 will not be included in nixos 25.11
- this release note entry should be moved to `rl-2605.section.md` (which will be created on 2025-11-23)